.asf.yaml

```diff
@@ -7,6 +7,49 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+github:
+  description: "Apache Geode"
+  homepage: https://geode.apache.org/
+  labels:
+    - geode
+    - datagrid
+    - apache
+  features:
+    # Enable wiki for documentation
+    wiki: false
+    # Enable issue management
+    issues: false
+    # Enable projects for project management boards
+    projects: true
+
+  enabled_merge_buttons:
+    # enable squash button:
+    squash:  true
+    # enable merge button:
+    merge:  true
+    # disable rebase button:
+    rebase:  true
```

Comment:
The comment for this flag seems at odds with what the flag is set to. Does a value of `true` means that the rebase button is disabled?

Reply:
Whoops, bad comment!